src/mux-analytics.brs

```diff
@@ -918,32 +950,32 @@ function muxAnalytics() as Object
       props.view_start = FormatJson(m._viewStartTimestamp)
     end if
     if m._viewTimeToFirstFrame <> Invalid AND m._viewTimeToFirstFrame <> 0
-      props.view_time_to_first_frame = m._viewTimeToFirstFrame.toStr()
+      props.view_time_to_first_frame = m._viewTimeToFirstFrame
     end if
     if m._contentPlaybackTime <> Invalid AND m._contentPlaybackTime <> 0
-      props.view_content_playback_time = m._contentPlaybackTime.toStr()
-      props.view_total_content_playback_time = m._contentPlaybackTime.toStr()
+      props.view_content_playback_time = m._contentPlaybackTime
+      props.view_total_content_playback_time = m._contentPlaybackTime
     end if
     if m._viewWatchTime <> Invalid AND m._viewWatchTime <> 0
-      props.view_watch_time = m._viewWatchTime.toStr()
+      props.view_watch_time = m._viewWatchTime
     end if
     if m._viewRebufferCount <> Invalid
       props.view_rebuffer_count = m._viewRebufferCount.toStr()
```

Comment:
Is there a reason to keep this as a string? From our side, it should be a number, so I'd rather keep it a number if possible.